openfun/fun-apps
# Ficus rebase: Remove course feed

Removes course feed functionality.

Cf. [OC-2940](https://tasks.opencraft.com/browse/OC-2940)

**Test instructions**

1. Access http://localhost:8000/cours/feed/. Observe that the page displays content similar to the following:

    ```
    <?xml version="1.0" encoding="utf-8"?>
    <rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>Derniers cours publiés sur FUN</title><link>http://example.com/cours/feed/</link><description>Derniers cours publiés sur www.fun-mooc.fr</description><atom:link href="http://example.com/cours/feed/" rel="self"></atom:link><language>fr</language><lastBuildDate>Fri, 11 Aug 2017 12:18:12 -0000</lastBuildDate></channel></rss>
    ````

2. Check out this branch.

3. Access http://localhost:8000/cours/feed/ again. Observe a 404.

**Reviewers**


CC @julAtWork
